src/utils.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * @license
+ * Copyright (c) 2015 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at
+ * http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at
+ * http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+export function camelCaseToKebab(camelCased: string): string {
```

Comment:
Done. Thanks for recommending this! Turns out StudlyCased names aren't converted to attributes at all!

Added a test to confirm and moved the function to be private to elements-generator